### PR TITLE
chore: update .snk file for strong naming

### PR DESCRIPTION
**Pull request by 96malhar**
*Issue #, if available:*
DOTNET-7248

*Description of changes:*
* Updated the `.snk` file that is used for strong naming.
* Verified that all projects build correctly and the resulting assemblies are strong-named with the expected public key token.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/AWS.SessionProvider.Net35.csproj [hunk 1]
```diff
@@ -24,18 +24,11 @@
...
-    </When>
-    <Otherwise>
-      <PropertyGroup>
-        <AssemblyOriginatorKeyFile>$(AWSKeyFile)</AssemblyOriginatorKeyFile>
-      </PropertyGroup>
-    </Otherwise>
-  </Choose>
+
+  <PropertyGroup>
+    <AssemblyOriginatorKeyFile>public.snk</AssemblyOriginatorKeyFile>
```
Comment (philasmar) [inline src/AWS.SessionProvider.Net35.csproj:29]: why don't we have `<SignAssembly>true</SignAssembly>` here?

File: src/AWS.SessionProvider.Net45.csproj [hunk 2]
```diff
@@ -20,18 +20,11 @@
...
-    </When>
-    <Otherwise>
-      <PropertyGroup>
-        <AssemblyOriginatorKeyFile>$(AWSKeyFile)</AssemblyOriginatorKeyFile>
-      </PropertyGroup>
-    </Otherwise>
-  </Choose>
+
+  <PropertyGroup>
+    <AssemblyOriginatorKeyFile>public.snk</AssemblyOriginatorKeyFile>
```
Comment (philasmar) [inline src/AWS.SessionProvider.Net45.csproj:25]: why don't we have `<SignAssembly>true</SignAssembly>` here?

File: src/AWS.SessionProvider.Net35.csproj (hunk 1 above)
Comment (96malhar) [reply src/AWS.SessionProvider.Net35.csproj:29]: It's there on line 25.

File: src/AWS.SessionProvider.Net45.csproj (hunk 2 above)
Comment (96malhar) [reply src/AWS.SessionProvider.Net45.csproj:25]: It's there on line 21

**Review by philasmar — APPROVED** [review]

**Pull request merged**